package.json

```diff
@@ -37,9 +38,10 @@
     "node": ">=0.4.x"
   },
   "dependencies": {
-    "underscore": "1.8.3",
     "mapnik-reference": "~6.0.2",
-    "optimist": "~0.6.0"
+    "optimist": "~0.6.0",
+    "torque-reference": "git+https://github.com/CartoDB/torque-reference.git",
```

Comment:
If you want to have it pointing to your working branch, use something like this: `"torque-reference": "cartodb/torque-reference#your-working-banch"`